README.md

```diff
@@ -7,6 +7,7 @@ Graphical Unix-like operating system for x86 computers.
 [![Fuzzing Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/serenity.svg)](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:serenity)
 [![Sonar Cube Static Analysis](https://sonarcloud.io/api/project_badges/measure?project=SerenityOS_serenity&metric=ncloc)](https://sonarcloud.io/dashboard?id=SerenityOS_serenity)
 [![Discord](https://img.shields.io/discord/830522505605283862.svg?logo=discord&logoColor=white&logoWidth=20&labelColor=7289DA&label=Discord&color=17cf48)](https://discord.gg/serenityos)
+[![justforfunnoreally.dev badge](https://img.shields.io/badge/justforfunnoreally-dev-9ff)](https://github.com/seven1m/justforfunnoreally.dev)
```

Comment:
Should the link point to the site, instead of the github repo? 

https://justforfunnoreally.dev/

Reply:
Hm probably. The part of the website that says "copy paste into your project readme" links to the github, but maybe that's just so that we can find the place to PR a table entry?

Reply:
Ahh, that should link to the website. My mistake!